frontend/javascripts/oxalis/view/right-border-tabs/segments_tab/segments_view.tsx

```diff
@@ -1136,6 +1511,24 @@ class SegmentsView extends React.Component<Props, State> {
     this.props.onUpdateSegmentGroups(newSegmentGroups, this.props.visibleSegmentationLayer.name);
   }
 
+  doesGroupHaveAnyMeshes = (groupId: number): boolean => {
+    const { visibleSegmentationLayer } = this.props;
+    if (visibleSegmentationLayer == null) return false;
+    const segmentGroup = this.getSegmentsOfGroup(groupId);
+    if (segmentGroup == null) return false;
+    let bool = false;
+    segmentGroup.forEach((segment) => {
+      if (
+        Store.getState().localSegmentationData[visibleSegmentationLayer.name].isosurfaces[
```

Comment:
you could extract `Store.getState().localSegmentationData[visibleSegmentationLayer.name].isosurfaces` into a variable (outside of the forEach loop) to save some look ups. I think, I saw the same pattern somewhere else, too..